apps/web/app/_api/mutations/useItemDraw.ts

```diff
@@ -0,0 +1,16 @@
+import { useMutation } from '@tanstack/react-query';
+
+import { fetcher } from '../fetcher';
+import { ItemDrawResponse } from '../types/draw';
+
+const itemDraw = (): Promise<ItemDrawResponse> => {
+  return fetcher.post<ItemDrawResponse>('items/draw', { json: '' });
+};
+
+export const useItemDraw = () => {
+  return useMutation<ItemDrawResponse, Error>({
+    mutationFn: () => {
+      return itemDraw();
+    },
+  });
```

Comment:
여기도 추후 어떻게 될 진 모르지만 지금은 이렇게 해도 되지 않나 생각해여

```suggestion
  return useMutation<ItemDrawResponse, Error>({
    mutationFn: itemDraw
  });
```